.travis.yml

```diff
@@ -9,10 +9,11 @@ branches:
     - /^\d+\.\d+(\.\d+)?(-\S*)?$/
     #hotfix branches (not being too strict on purpose)
     - /^hf-.*$/
+    - exp/mutationTesting-pimp
 
 script:
   - mvn test -B
-  - mvn org.pitest:pitest-maven:mutationCoverage
+  - mvn pitmp:run
```

Comment:
my bad, but this seems to be the way to run pit on multi modules so that we can ignore some modules (for instance openml-example module). But I'm finding some strange results (https://github.com/STAMP-project/pitmp-maven-plugin/issues/17)